debian/rules

```diff
@@ -41,6 +41,11 @@ endif
 # control, so disable it on non-Ubuntu builds
 ifeq ($(filter Ubuntu,$(DEB_VENDOR)),)
 	COMMON_CONFIGURE_OPTIONS += -DMIR_RUN_WLCS_TESTS=OFF
+else
+  # Disable wlcs tests on Launchpad armhf, arm64 (MirServer/mir#2474)
+  ifneq ($(filter armhf arm64,$(DEB_HOST_ARCH)),)
+    COMMON_CONFIGURE_OPTIONS += -DMIR_RUN_WLCS_TESTS=OFF
+  endif
```

Comment:
If at all, we should only disable them when cross-building - which is where we see those failures.

But let's see where does #2490 take us.